atlasdb-console/src/main/groovy/com/palantir/atlasdb/console/module/AtlasCoreModule.groovy

```diff
@@ -50,35 +50,62 @@ class AtlasCoreModule implements AtlasConsoleModule {
                          Ex: def x = table('stream_value')
 
                          To retrieve metadata about a table, use <TABLE>.describe(),
-                         <TABLE>.isDynamic(), <TABLE>.rowComponents(), and <TABLE>.columnNames().'''.stripIndent(),
+                         <TABLE>.isDynamic(), <TABLE>.rowComponents(), and <TABLE>.columnNames().
+                         '''.stripIndent(),
+
             'transactions': '''\
                          Atomic transactions used to avoid read/write or write/write
```

Comment:
This statement doesn't make any sense. Seems like the person who wrote it didn't understand what a transaction is.